src/main/webapp/resources/js/components/project/ProjectSelect.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import { Select, SelectProps, Tag, Typography } from "antd";
+import { LabeledValue } from "antd/lib/select";
+
+export type Project = { identifier: number; name: string };
+
+export interface ProjectSelectProps extends SelectProps {
+  projects: Project[];
+}
+
+/**
+ * React component for displaying a project drop-down menu.
+ * @param projects - list of projects that is to be displayed
+ * @param onChange - function that is called when select option has changed
+ * @param defaultValue - project identifier of the project that is to be displayed by default
+ * @constructor
+ */
+export function ProjectSelect({
+  projects,
+  onChange = () => {},
```

Comment:
I would just not set a default value for the `onChange` since it really cannot function without one.

Reply:
Please see 8023bed5bf9acf057c2433e225379d0c850cc7fe.